### DISCUSSION TITLE
[Access control] Is it possible to make a video visible only to certain users ?

### DISCUSSION THREAD

**Discussion by shodanx2** [mediacms-io/mediacms#521] [Q&A]
So, question about access control

I would like to make some videos visible to all, but some videos only visible to logged in users and only certain users or group of users ? 

Can that be done with mediacms ? 

Is there any way to create a user group ?

**Comment by mgogoulos** [comment]
Hi, this is very specific behavior and the software does not allow it out of the box (however it wouldn't be very hard to develop, if you have the resources). Allowing only logged-in users though is something that is supported: https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#523-configure-only-member-access-to-media
